### PR TITLE
Basic avoid basic trap

### PR DESCRIPTION
**Pull request by Antimatter543**
probably be issues with bomb_tracker

### CODE REVIEW

**Comment by Antimatter543** [comment]
Maybe not approve until I add the trapping mechanisms + fix conflicts (although if you need the avoid_bomb strat, go ahead).

File: python3/totoro_agent/utils/util_functions.py [hunk 1]
```diff
@@ -264,13 +264,16 @@ def get_blast_zone(bomb_loc, diameter, entities, world):
         if is_in_bounds(tile, world_width, world_height) and entity_at(tile, entities) not in block_tile:
             blast_tiles.append(tile)
         else:
+            cur_loc = bomb_loc
```
Comment (chrisrabe) [inline python3/totoro_agent/utils/util_functions.py:267]: can remove this

**Review by chrisrabe — CHANGES_REQUESTED** [review]
get_blast_zone has bugs. Also no need to create a new key for your strategy. Can just replace what's in retreat

File: python3/totoro_agent/brain/brain.py [hunk 2]
```diff
@@ -31,20 +32,30 @@ def get_next_strategy(self, game_state) -> str:
...
+
+
+
+
 
         # If you're in the blast tiles, do RETREAT
         if game_state['player_pos'] in game_state['all_hazard_zones'] or game_state['player_on_bomb']:
             print('HOLY RUN FOR YOUR LIFE YOU ARE GONNA GET RAILED')
-            return 'retreat'
+            return 'basic_avoid' # Basic avoid vs retreat. Retreat value based, basic avoid is coded.
```
[the remark this answers is not in the document]
Comment (Antimatter543) [reply python3/totoro_agent/brain/brain.py:55]: Got it. I'll copy paste your blast_zone func from develop into this branch once at home.

**Review by chrisrabe — APPROVED** [review]

**Pull request merged**